src/site/markdown/server/manage-webadmin.md

```diff
@@ -436,6 +436,27 @@ Note that several calls to this endpoint will be run in a sequential pattern.
 
 If the server restarts during the migration, the migration is silently aborted.
 
+## Correcting ghost mailbox
+
+This is a temporary workaround for the **Ghost mailbox** bug encountered using the Cassandra backend, as described in MAILBOX-322.
+
+You can use the mailbox merging feature in order to merge the old "ghosted" mailbox with the new one.
+
+```
+curl -XPOST http://ip:port/cassandra/mailbox/merging -d `{"mergeOrigin":"id1", "mergeDestination":"id2"}`
+```
+
+Will:
+ - Delete references to `id1` mailbox
+ - Move it's messages into `id2` mailbox
+ - Union the rights of both mailboxes
```

Comment:
Could be great to give more explanation about how you UNION right maybe

Reply:
Moreover I think it would be better to take the right of the last visible one if we can determine which one it is

Reply:
👎  

We don't want to loose user actions. Hence union seems better fitted.

Take OP.LNG, ghost mailbox is around for months. Don't you think users might have shared their mailboxes with other peoples? Don't you think we should not loose that information?

Reply:
UNION is btw a basic mathematical ensemblists operation. (for ACL and for Rights). I don't know what can be explained more about it.

Reply:
ok